kratos/tests/auxiliar_files_for_python_unittest/materials_files/initial_time_First.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I'd rename the file to `initial_time_first.json` to follow our snake case filename style guide.

Reply:
done